sql/item_mods.sql

```diff
@@ -21567,9 +21562,6 @@ INSERT INTO `item_mods` VALUES (19716,27,16);
 INSERT INTO `item_mods` VALUES (19716,161,-16);
 INSERT INTO `item_mods` VALUES (19716,288,25);
 INSERT INTO `item_mods` VALUES (19716,302,5);
-INSERT INTO `item_mods` VALUES (19717,30,30);

```

Comment:
If you have new liens for the same item, add them here. Don't go trying to sort the list by groups of "types" of item, like jamming all mythic in one spot.